enhancements/aws-pod-identity.md

```diff
@@ -0,0 +1,442 @@
+---
```

Comment:
also this needs to pick a domain ??

Reply:
you mean an subdirectory under enhancements? i'm open to suggestions :+1:

Reply:
I'm working on a CCO enhancement as well, my best guess was security.